src/hooks/useCourtContracts.js

```diff
@@ -38,17 +38,17 @@ import tokenAbi from '../abi/ERC20.json'
 import votingAbi from '../abi/CRVoting.json'
 
 const GAS_LIMIT = 1200000
-const ANJ_ACTIVATE_GAS_LIMIT = 500000
-const ANJ_ACTIONS_GAS_LIMIT = 325000
+const ANT_ACTIVATE_GAS_LIMIT = 500000
+const ANT_ACTIONS_GAS_LIMIT = 325000
 const ACTIVATE_SELECTOR = getFunctionSignature('activate(uint256)')
 
-// ANJ contract
-function useANJTokenContract() {
-  const { anjToken } = useCourtConfig()
+// ANT contract
+function useANTTokenContract() {
+  const antToken = getANTToken()
```

Comment:
We should still be able to get the ANT token from the config
```suggestion
  const { token: antToken } = useCourtConfig()
```

Reply:
ah this one right? <https://github.com/aragon/protocol/blob/f1b3361a160da92b9bb449c0a05dee0c30e41594/packages/subgraph/schema.graphql#L3>